### ISSUE TITLE
[Bug]: Encoding breaks when exporting the model

### ISSUE THREAD

**Issue by LukasKStavario** [IFCjs/web-ifc#440]
### What happened?

I am trying hard to export my models, according to #435 I have tested about 50 models and only 3 I can export.

But If I export model with some specific czech coding, the coding breaks the model for other viewers and breaks the string itself

![image](https://github.com/IFCjs/web-ifc/assets/112091369/1c014ded-5aa0-41eb-b749-47b448232794)

According to image your algorhitm changes \X2 to \X and 00E9 to E9 which is not correct by the structure.
And it is done in whole file
![image](https://github.com/IFCjs/web-ifc/assets/112091369/e9052541-5dcc-47b3-9e95-76193cf234cd)

![image](https://github.com/IFCjs/web-ifc/assets/112091369/3349bcde-ac77-452a-b336-5f9d65700d03)

  

### Version

0.0.42

### What browsers are you seeing the problem on?

Chrome

**Comment by beachtom** [comment]
I am really a bit baffled as to how \X is being introduced - as we do not use that anywhere. Can you indicate a model that this breaks for (it may be one I already have) and are you just opening it and saving it?

**Comment by LukasKStavario** [comment]
Yes bug appears on models that I have sent before (on export #441 issue). Encoded text is in original "Označení položky". \X was example, the encoding is totaly breaked. If I can help any further do not hesitate to contact me.

**Comment by LukasKStavario** [comment]
Any news on this?

**Comment by beachtom** [comment]
Not yet I am afraid - still trying to work out why it is breaking

**Comment by beachtom** [comment]
I now know what the issue is - I just need to fix it

**Comment by beachtom** [comment]
This should now be fixed
